docs/content/integrations/fivetran/fivetran.mdx

```diff
@@ -0,0 +1,192 @@
+
+```python file=/integrations/fivetran/sync_and_materialize_fivetran_assets.py
+from dagster_fivetran import (
+    FivetranWorkspace,
+    build_fivetran_assets_definitions,
+    fivetran_assets,
+)
+
+import dagster as dg
+
+fivetran_workspace = FivetranWorkspace(
+    account_id=dg.EnvVar("FIVETRAN_ACCOUNT_ID"),
+    api_key=dg.EnvVar("FIVETRAN_API_KEY"),
+    api_secret=dg.EnvVar("FIVETRAN_API_SECRET"),
+)
+
+all_fivetran_assets = build_fivetran_assets_definitions(fivetran_workspace)
+
+defs = dg.Definitions(
+    assets=all_fivetran_assets,
+    resources={"fivetran": fivetran_workspace},
+)
+```
+
+### Customize the materialization of Fivetran assets
+
+You can use the <PyObject module="dagster_fivetran" method="fivetran_assets" /> decorator to create the assets definition for a given connector. Using this approach, you can customize the code within your `@fivetran_assets`-decorated function.
+
+```python file=/integrations/fivetran/customize_fivetran_asset_defs.py
+from dagster_fivetran import (
+    FivetranWorkspace,
+    build_fivetran_assets_definitions,
+    fivetran_assets,
+)
+
+import dagster as dg
+
+fivetran_workspace = FivetranWorkspace(
+    account_id=dg.EnvVar("FIVETRAN_ACCOUNT_ID"),
+    api_key=dg.EnvVar("FIVETRAN_API_KEY"),
+    api_secret=dg.EnvVar("FIVETRAN_API_SECRET"),
+)
+
+
+@fivetran_assets(
+    connector_id="fivetran_connector_id",
+    name="fivetran_connector_id",
+    group_name="fivetran_connector_id",
+    workspace=fivetran_workspace,
+)
+def fivetran_connector_assets(
+    context: dg.AssetExecutionContext, fivetran: FivetranWorkspace
+):
+    # Do something before the materialization...
+    yield from fivetran.sync_and_poll(context=context)
+
+
+defs = dg.Definitions(
+    assets=[fivetran_connector_assets],
+    resources={"fivetran": fivetran_workspace},
+)
+```
+
+### Customize asset definition metadata for Fivetran assets
+
+By default, Dagster will generate asset specs for each Fivetran asset and populate default metadata. You can further customize asset properties by passing an instance of the custom <PyObject module="dagster_fivetran" object="DagsterFivetranTranslator" /> to the <PyObject module="dagster_fivetran" method="load_fivetran_asset_specs" /> function.
+
+```python file=/integrations/fivetran/customize_fivetran_translator_asset_spec.py
+from dagster_fivetran import (
+    DagsterFivetranTranslator,
+    FivetranConnectorTableProps,
+    FivetranWorkspace,
+    load_fivetran_asset_specs,
+)
+
+import dagster as dg
+
+fivetran_workspace = FivetranWorkspace(
+    account_id=dg.EnvVar("FIVETRAN_ACCOUNT_ID"),
+    api_key=dg.EnvVar("FIVETRAN_API_KEY"),
+    api_secret=dg.EnvVar("FIVETRAN_API_SECRET"),
+)
+
+
+# A translator class lets us customize properties of the built
```

Comment:
I thought we agreed on using replace_attributes / map_asset_specs for this?

Reply:
That makes sense - will add that in a separate PR